test/paths.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"bufio"
+	"os"
+	"strings"
+)
+
+// GetRelativeDependencyPath returns the relative path for the specified module name
+//
+// moduleName string - module name or substring
+//
+// return string
+func GetRelativeDependencyPath(moduleName string) string {
+	var CRDPath string
```

Comment:
Rename it to `crdPath` to be consistent with Go variables naming.

Reply:
fixed